### PR TITLE
Update image tags for version 3.0.0

### PR DESCRIPTION
**Pull request by ymrsmns**
Upgrade image tags

#### PR Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] Variables and other changes are documented in the README.md

**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

* Update values.yaml
* Update Chart.yaml
* Update version in README.md

### CODE REVIEW

File: charts/goldpinger/Chart.yaml [hunk 1]
```diff
@@ -10,4 +10,5 @@ maintainers:
 name: goldpinger
 sources:
   - https://github.com/bloomberg/goldpinger
-version: 2.0.2
+  - https://github.com/okgolove/helm-charts
+version: 2.0.3
```
Comment (okgolove) [inline charts/goldpinger/Chart.yaml:14]: I think we should bump major version here.

**Comment by okgolove** [comment]
Hi @ymrsmns. Thank you for the PR.
Could you rebase you branch and bump major chart's version?

File: charts/goldpinger/Chart.yaml (hunk 1 above)
Comment (ymrsmns) [reply charts/goldpinger/Chart.yaml]: ok. updated version charts

**Comment by okgolove** [comment]
@ymrsmns steel need to resolve merge conflicts.
Could you rebase/merge and resolve conflicts?

**Comment by ymrsmns** [comment]
Fixed the resulting conflict

**Pull request merged**